OpenSearch.sln

```diff
@@ -12,10 +12,10 @@ EndProject
 Project("{F2A71F9B-5D33-465A-A702-920D77279786}") = "scripts", "build\scripts\scripts.fsproj", "{D6997ADC-E933-418E-831C-DE1A78897493}"
 EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "tests", "tests", "{6C4A2627-AF22-4388-9DF7-7A9AEACFD635}"
-ProjectSection(SolutionItems) = preProject
```

Comment:
What's the purpose to these whitespace changes in this file?

Reply:
These were done by `dotnet` cli. 

It looks like `ProjectSection` is an element under `Project` and so needs an indent.